config/samples/backends/lvm-nvme-tcp/backend.yaml

```diff
@@ -0,0 +1,27 @@
+# This sample requires an LVM VG with the name of `cinder-volumes` to exist on the OpenShift node where cinder-volume runs
+# It also requires that nvme-fabrics kernel module is running everywhere and that nvmet is running on the controller host
+# The correct way to configure the load of additional kernel modules is using `MachineConfig` as shown in `nvme-fabrics.yaml`.yaml`.
+# To create the LVM on CRC the sample also uses a `MachineConfig` using a systemd unit with file `lvm.yaml`
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        lvm-nvme-tcp:
+          customServiceConfig: |
+            [lvm]
+            volume_backend_name=lvm_nvme_tcp
+            volume_driver=cinder.volume.drivers.lvm.LVMVolumeDriver
+            target_helper=nvmet
+            target_protocol=nvmet_tcp
+            target_port=4420
+            target_prefix=nvme-subsystem-1
+            nvmeof_conn_info_version=2
+            image_volume_cache_enabled=false
+            use_multipath_for_image_xfer=false
```

Comment:
Why false?

Reply:
Because NVMe-oF multipathing (ANA) is different from the multipathd approach, where I know that a single path works just fine. So until we have an isolated networks example with 2 networks and configuring the driver to do multipathing I'd rather not enable it.

Reply:
Sure, that makes sense. It means we need to walk a fine line between these sample files being intended to document things for the user, versus the current state of our own development.

Is this an instance where a comment would help? Maybe something that states, "Disable multipath until it can be tested," or "TODO: enable multipath after it can be tested" (and you know I'm not a fan of seeing too many TODO in the code!)

Reply:
I can add a comment with something like:
```
# LVM NVMe-oF multipathing (ANA) requires special configuration to have multiple networks
# accessible from compute nodes and other cinder controller pods.
```

Reply:
I'm OK leaving this as-is for now, and I think we can do a global tweak/cleanup when we get closer to GA.

Reply:
OK